server/data/data-file/src/main/java/org/apache/james/mailrepository/file/MBoxMailRepository.java

```diff
@@ -498,16 +484,12 @@ public boolean isComplete() {
                 }
 
                 public MimeMessage messageAction(String messageSeparator, String bodyText, long messageStart) {
-                    try {
-                        String key = generateKeyValue(bodyText);
-                        mList.put(key, messageStart);
-                        if ((LOGGER.isDebugEnabled())) {
-                            LOGGER.debug(this.getClass().getName() + " Key " + key + " at " + messageStart);
-                        }
-
-                    } catch (NoSuchAlgorithmException e) {
-                        LOGGER.error("MD5 not supported! ", e);
+                    String key = generateKeyValue(bodyText);
+                    mList.put(key, messageStart);
+                    if ((LOGGER.isDebugEnabled())) {
+                        LOGGER.debug(this.getClass().getName() + " Key " + key + " at " + messageStart);
```

Comment:
``LOGGER.debug("Key {} at {}", key, messageStart)``
You don't need to specify the class name because it will be already be added by sl4j automatically I think

Reply:
two commits added for more cleanup